CHANGELOG.rst

```diff
@@ -7,6 +7,11 @@ Development Version:
   By `Kai Mühlbauer <https://github.com/kmuehlbauer>`_.
 - Create/attach dimension scales on the fly, instead of at sync/flush (file-close).
   By `Kai Mühlbauer <https://github.com/kmuehlbauer>`_.
+- Ensure order tracking is true for newly created netcdf4 files as required
+  by the netcdf4 standard. A new keyword argument has been added to the class
+  constructor ``h5netcdf.File``, ``track_order`` to control the feature
+  for newly created files. (Closes Issue #128)
```

Comment:
```suggestion
  for newly created files. This enables files created by h5netcdf to be
  appended to by netCDF4 library users. (Closes Issue #128)
```
Do we still have time to add this clarification?

Reply:
Sure!